File: jules-scratch/verification/verify_qr_code.ts
import { test, expect } from '@playwright/test';

test('basic test', async ({ page }) => {
  await page.goto('http://localhost:3000/neoframe.html');
  await page.getByLabel('Enable QR Code').check();
  await expect(page.locator('canvas#qr-code-canvas')).toBeVisible();
  await page.screenshot({ path: 'jules-scratch/verification/verification.png' });
});

File: jules-scratch/verification/verify_qr_code.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch(args=['--no-sandbox'])
    page = browser.new_page()
    page.goto("http://localhost:3000/neoframe.html")
    page.screenshot(path="jules-scratch/verification/verification.png")
    browser.close()